src/storage/azure.js

```diff
@@ -37,19 +37,42 @@ const downloadFileAsJson = ({ blobServiceClient }) => async (
   bucketName,
   fileName
 ) => {
-  const downloadResponse = await blobServiceClient
-    .getContainerClient(bucketName)
-    .getBlockBlobClient(fileName)
-    .download(0);
   try {
-    return await streamToString(
-      downloadResponse.readableStreamBody
-    ).then((data) => JSON.parse(data));
+    return await downloadFileAsString({blobServiceClient})(bucketName, fileName)
+        .then((data) => JSON.parse(data));
   } catch (e) {
     return Promise.reject("File is not in JSON format");
   }
 };
 
+const downloadFileAsString = ({ blobServiceClient }) => async (
+    bucketName,
+    fileName
+) => {
+    const downloadResponse = await blobServiceClient
+        .getContainerClient(bucketName)
+        .getBlockBlobClient(fileName)
+        .download(0);
+    try {
+        return await streamToString(
+            downloadResponse.readableStreamBody
+        )
+    } catch (e) {
+        return Promise.reject("Could not stringify blob");
+    }
+};
+
+const getBlobProperties = ({ blobServiceClient }) => async (
+    bucketName,
+    fileName
+) => {
+    const properties = await blobServiceClient
```

Comment:
you can just return the result, at which case you'd be able to drop the brackets and return (eslint `arrow-body-style`)